frontend/src/lib/k8s/endpoints.ts

```diff
@@ -44,7 +44,7 @@ class Endpoints extends makeKubeObject<KubeEndpoint>('endpoint') {
   }
 
   getAddressesText() {
-    this.getAddresses().join(', ');
+    return this.getAddresses();
```

Comment:
If we are just sending the addresses, then we don't need this method. This method should be fixed to do what it's supposed to be doing (which is the fix that @Src0p had).